dss-pades/src/test/java/eu/europa/esig/dss/pades/signature/suite/PAdESLevelBSignWithFileMemoryUsageSettingTest.java

```diff
@@ -0,0 +1,227 @@
+package eu.europa.esig.dss.pades.signature.suite;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
There is commented code that I left waiting your advice that's because...

Please have a look at the other comment about 'unit testing' the pdf libraries.